packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -275,18 +255,23 @@ class ChartController extends DisposableController
 
   String title;
 
-  displayDuration zoomDuration = displayDuration.live;
+  /// zoomDuration values of:
+  ///     null implies all (default)
+  ///     Duration() imples live
+  ///     Duration(minutes: 5) implies 5 minute interval
+  ///     Duration(minutes: 15) implies 15 minute interval
+  Duration _zoomDuration = const Duration();
```

Comment:
Seems like live is the default not null.

Reply:
Oops comment matches reality.